### PR TITLE
#76 Run command for tvac programs

### PR DESCRIPTION
**Pull request by cowen-mawson**
This adds a "run" command to `uts-cli` which will take a program name and look for a toml file with that name from the current directory.

The command execs `uts-run` which it expects to find in the current directory.

### CODE REVIEW

**Comment by cowen-mawson** [comment]
It was going to need a refactoring to be able to call out to the program runner from uts-cli as an internal call as the API from different binary crates in rust aren’t visible to each other, so I took a quicker option of calling out to the uts-run binary. This does mean that the new run command expects to find uts-run in the same directory. I hope that doesn’t complicate distribution for the tvac test.

File: hestia-sat/src/bin/uts-cli/main.rs [hunk 1]
```diff
@@ -122,6 +132,28 @@ fn do_max(board: u8, temp: f32) {
     show_status(board);
 }
 
+fn do_run(program_name: &str) {
+    let mut path_to_bin = env::current_dir().expect("Could not get current dir");
```
[the remark this answers is not in the document]
Comment (mryall-mawson) [reply hestia-sat/src/bin/uts-cli/main.rs]: As discussed, I've moved the program logic around. Have pushed to this branch and will update the PR now (if I can).

File: hestia-sat/.idea/runConfigurations/Test.xml
Comment (mryall-mawson) [inline hestia-sat/.idea/runConfigurations/Test.xml:1]: This runs the unit tests on Mac. Probably should document this at some point, but run configuration will be enough for now.

**Review by mryall-mawson — APPROVED** [review]
Updated with changes as discussed

**Comment by mryall-mawson** [comment]
I'm going to merge this without having tested it on a real board yet, just to keep the branched code to a minimum.

**Pull request merged**